docs/source/model_doc/bertweet.rst

```diff
@@ -10,7 +10,7 @@
     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
     specific language governing permissions and limitations under the License.
 
-Bertweet
+BERTweet
```

Comment:
This should be done in separate PR

Reply:
BERTweet & PhoBERT are also my models, that why I made some changes to them.